packages/gatsby-cli/package.json

```diff
@@ -40,7 +40,7 @@
     "react": "^16.8.0",
     "redux": "^4.0.5",
     "resolve-cwd": "^3.0.0",
-    "semver": "^6.3.0",
+    "semver": "^7.3.2",
```

Comment:
This one is tricky - because global cli is meant to be as backward compatible - this move will break newer cli versions (as semver@7 needs `node>=10`) for folks trying to use older gatsby versions (that are running fine on node 8 - older v2 versions or even node 6 - gatsby v1)

Reply:
Maybe I misunderstood but: https://www.gatsbyjs.org/docs/basic-hardware-software-requirements/#nodejs-ecosystem

Let me know what you want to do here; I don't think this is unreasonable as the wider ecosystem shifts away from unsupported stuff but this a wider discussion point that Gatsby needs support

Reply:
Other packages in gatsby-cli that also need major updates (and that require node 10 also): chalk, execa, fs-extra, semver

Reply:
You probably are right. Older gatsby-cli versions will work fine on older node versions, so upgrade that bumps node version should fine